services/api/handlers/projects.js

```diff
@@ -254,6 +254,27 @@ exports.patch = {
   }
 };
 
+exports.delShallow = function(request, reply) {
+  request.server.methods.projects.remove(
+    [
+      request.params.project
+    ],
+    function(err, result) {
+      if(err) {
```

Comment:
you need to write a similar test to that which triggers the error condition in this callback I think.